### ISSUE TITLE
STB xyz convert uv coordinat is false

### ISSUE THREAD

**Issue by gongzaizhou** [lmb-freiburg/hand3d#33]
I try your create_db.m, I find uv coordinate is false in 'BB', how should I do?

**Comment by zimmerm** [comment]
Can you explain your problem in more detail? I find it hard to follow your problem.

**Comment by KennethJAnthony** [comment]
Hi @zimmerm, 

can I check with you for create_db.m it will create a binary of the 6000 * 2 images in B1Random & B1Counting. This will create a 5.5 GB file. And this is the file that is used to evaluate with eval_full.py.

**Comment by KennethJAnthony** [comment]
Hi,

For training_lifting.py which variant was used to trained the weights in the paper. I am trying to replicate the results for STB dataset.
